doc/release_notes/release_notes_3.0.rst

```diff
@@ -50,7 +69,14 @@ that is essential to build ACRN. Compared to previous versions, ACRN v3.0 adds
 the following hardware information to board XMLs to support new features and
 fixes.
 
-  - TBD
+  - Add a cmdline option to inject LLC CAT capability
```

Comment:
Can you specify what tool this was added to (in this case, the `board_inspector` tool), and add a link to that user manual (using `:ref:`)

Reply:
Updated.